recipes/kpop/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "kpop" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/PaoloRibeca/KPop/releases/download/v{{ version }}/KPop-{{ version }}-Linux_x86-64.tar.xz  # [linux]
+    sha256: 915eb6d4e0b41b4f6409b5fd97188193106b3300734c92131ab8ad0618051e4a  # [linux]
+  - url: https://github.com/PaoloRibeca/KPop/releases/download/v{{ version }}/KPop-{{ version }}-MacOS_x86-64.tar.xz  # [osx]
+    sha256: 06c15129e9ecea58092f5e4d5df136a0c227815f011f80f63ad9093d70cc46d3  # [osx]
```

Comment:
I see your point. However, BiOCamLib is included in KPop sources as a Git submodule, so there would never be a dependency problem when recompiling KPop from source (and many more packages would need to be installed when compiling from source). We actually include BiOCamLib as a dependency because it provides some executables that are/will be needed to run tests/examples. It makes sense to specify which version of BiOCamLib KPop depends on though, to avoid the possible API compatibility problems that you mention - will do so now, and that should be sufficient

Reply:
Also, I do realize now that there are a couple of R dependency missing - I'll add them too